tempesta_fw/t/unit/test_sched_hash.c

```diff
@@ -43,96 +43,90 @@
 #include "sched_helper.h"
 #include "test.h"
 
-char req_str1[] = "GET / HTTP/1.1\r\nhost:host1\r\n\r\n";
-char req_str2[] = "GET / HTTP/1.1\r\nhost:host2\r\n\r\n";
-char req_str3[] = "GET / HTTP/1.1\r\nhost:host3\r\n\r\n";
-char req_str4[] = "GET / HTTP/1.1\r\nhost:host4\r\n\r\n";
-char *req_strs[] = {
-	req_str1,
-	req_str2,
-	req_str3,
-	req_str4,
+static char *req_strs[] = {
+	"GET / HTTP/1.1\r\nhost:host1\r\n\r\n",
+	"GET / HTTP/1.1\r\nhost:host2\r\n\r\n",
+	"GET / HTTP/1.1\r\nhost:host3\r\n\r\n",
+	"GET / HTTP/1.1\r\nhost:host4\r\n\r\n",
 };
-size_t req_strs_size = sizeof(req_strs) / sizeof(req_strs[0]);
 
-TEST(tfw_sched_hash, sg_empty)
+static TfwMsg *sched_hash_get_scheduler_arg(size_t connection_type);
+
+static void
+sched_hash_free_scheduler_arg(TfwMsg *msg)
 {
-	int i, j;
+	test_req_free((TfwHttpReq *)msg);
+}
 
-	TfwSrvGroup *sg = test_create_sg("test", "hash");
+static struct TestSchedHelper sched_helper_hash = {
+	.sched = "hash",
+	.conn_types = ARRAY_SIZE(req_strs),
+	.get_sched_arg = &sched_hash_get_scheduler_arg,
+	.free_sched_arg = &sched_hash_free_scheduler_arg,
```

Comment:
Isn't this some sort of tauthology? `sched_hash` is the common part of the name, so we know that it's related to a scheduler, and moreover, to the hash scheduler. So when we want to get/free an arg, what's the point in saying that it's a `scheduler_arg`? Just `sched_had_get/free_arg` would be sufficient, and make the name much shorter and concise.